src/components/map/map.tsx

```diff
@@ -0,0 +1,57 @@
+import 'leaflet/dist/leaflet.css';
+import { Icon, layerGroup, Marker } from 'leaflet';
+import { useRef, useEffect } from 'react';
+import { useMap } from '../../hooks/use-map';
+import { City, Offer } from '../../types/offers';
+import { UrlMarkers } from '../../const';
+
+type MapProps = {
+  city: City;
+  points: Offer[];
+  selectedOffer: Offer | undefined;
```

Comment:
@Ve1heora А можно тут написать вот так? Или что-то сильно изменится?

```suggestion
  selectedOffer?: Offer;
```